github/ci/prow-deploy/files/jobs/kubevirt/common-templates/common-templates-periodics.yaml

```diff
@@ -54,3 +54,58 @@ periodics:
       resources:
         requests:
           memory: "8Gi"
+- name: periodic-common-templates-nightly-centos-container-disk-image-builder
+  cron: "30 2 * * *"
+  annotations:
+    testgrid-create-test-group: "false"
+  decorate: true
+  decoration_config:
+    timeout: 1h
+    grace_period: 5m
+  max_concurrency: 1
+  labels:
+    preset-dind-enabled: "true"
+    preset-docker-mirror-proxy: "true"
+    preset-kubevirtci-quay-credential: "true"
+  extra_refs:
+    - org: kubevirt
+      repo: kubevirtci
+      base_ref: master
+      work_dir: true
+    - org: kubevirt
+      repo: common-templates
+      base_ref: containerdisk-automation
+  spec:
+    nodeSelector:
+      type: bare-metal-external
+    containers:
+    - image: quay.io/kubevirtci/bootstrap:v20210126-a12b6c0
+      command:
+        - "/usr/local/bin/runner.sh"
+        - "/bin/bash"
+        - "-c"
+        - |
+          apt-get update && apt-get install -y --no-install-recommends --no-upgrade expect ansible intltool libosinfo-1.0 libssl-dev osinfo-db-tools python-gi && rm -rf /var/lib/apt/lists/*
```

Comment:
There's a lot of duplication here, could you please extract all the code in both jobs to a script in common templates? 

As I understand it the only difference is the `_new_image.sh` script to call, you could set a `TARGET_OS` env var in the job definition (fedora or centos) and then call `./${TARGET_OS}_new_image.sh` from the new script in common-templates, wdyt?

Reply:
Btw. why you are not using this image, which already has everything installed https://github.com/kubevirt/project-infra/blob/master/images/kubevirt-infra-bootstrap/Dockerfile? I know there was whole infrastructure which builded this image which @danielBelenky did (I don't know in which state it is now).

Reply:
@ksimon1 : That image had an older version of docker that was complaining whil I was trying to add a docker insecure registry, so I reverted back to the bootstrap image. But I see that infra-bootstrap map also be updated. Let me check if it works now.

Reply:
@fgimenez : Moved the commands into a script in common-templates.

Reply:
@ksimon1 : Using the kubevirt-infra-bootstrap base image.